release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -2807,6 +2807,56 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, Operator group reconciliation in {olmv0-first} triggered unnecessary `ClusterRole` updates because of the changing order of aggregation rule selectors. As a result, unnecessary API server writes occurred. With this release, a bug fix ensures the deterministic order of `ClusterRoleSelectors` in the aggregation rule, reducing unnecessary API server writes and improving cluster stability. (link:https://issues.redhat.com/browse/OCPBUGS-57279[OCPBUGS-57279])
+
+* Previously, ignoring the `AdditionalTrustBundlePolicy` in assisted-service's installation configuration caused users to incorrectly set the `AdditionalTrustBundlePolicy`, affecting Federal Information Processing Standard (FIPS) and other installation configuration overrides. With this release, the installation configuration includes an `AdditionalTrustBundlePolicy` field, ensuring overrides are applied correctly. Users can correctly set the `AdditionalTrustBundlePolicy` field, and ensure that FIPS and other installation configuration overrides function as intended. (link:https://issues.redhat.com/browse/OCPBUGS-57208[OCPBUGS-57208])
+
+* Previously, the authentication process for the `/metrics` endpoint was missing a token review check and caused unauthorized requests. As a result, the {product-title} console was prone to `TargetDown` alerts. With this release, the token review for unauthorized requests occurs with the user token provided in the request context. As a result, unauthorized requests to the {product-title} console do not cause `TargetDown` alerts. (link:https://issues.redhat.com/browse/OCPBUGS-57180[OCPBUGS-57180])
+
+* Previously, the `Started` column was hidden because of reduced screen size. This caused the `VirtualizedTable` component to malfunction because of a missing sort function. As a result, the table sorting functionality was affected in the `PipelineRun` list pages. With this release, the table component handles missing sort functions correctly for reduced screen sizes. (link:https://issues.redhat.com/browse/OCPBUGS-57110[OCPBUGS-57110]) 
+
+* Previously, users did not configure a default masthead logo when customizing the theme. This omission caused an inconsistency in the user interface. With this release, the masthead logo displays a default option for both light and dark themes, improving the interface consistency. (link:https://issues.redhat.com/browse/OCPBUGS-57054[OCPBUGS-57054])
+
+* Previously, the cluster installation failed because of an invalid security group configuration for  a Network Load Balancer (NLB). This failure prevented the traffic from both primary subnets for bootstrapping. With this release, the security group is updated to allow traffic from both primary subnets for bootstrapping, and the cluster installation does not fail because of security group restrictions on additional primary subnets. (link:https://issues.redhat.com/browse/OCPBUGS-57039[OCPBUGS-57039])
```

Comment:
```suggestion
* Previously, the cluster installation failed because of an invalid security group configuration for a Network Load Balancer (NLB). This failure prevented the traffic from both primary subnets for bootstrapping. With this release, the security group allows traffic from both primary subnets for bootstrapping, and the cluster installation does not fail because of security group restrictions on additional primary subnets. (link:https://issues.redhat.com/browse/OCPBUGS-57039[OCPBUGS-57039])
```